src/html-entities.js

```diff
@@ -0,0 +1,309 @@
+/* 
+Copyright (c) 2015, Yahoo Inc. All rights reserved.
+Copyrights licensed under the New BSD License.
+See the accompanying LICENSE file for terms.
+
+Authors: Nera Liu <neraliu@yahoo-inc.com>
+         Albert Yu <albertyu@yahoo-inc.com>
+         Adonis Fung <adon@yahoo-inc.com>
+*/
+/*jshint -W030 */
+/*jshint eqnull:true */
+(function () {
+"use strict";
+
+var fs = require('fs');
+/* polyfill for codePointAt, fromCodePoint */
+require('./polyfill.js');
```

Comment:
check out the grunt file and you'll see polyfill.js will be included only for building dist/...js